flexmeasures/data/schemas/sensors.py

```diff
@@ -238,10 +246,28 @@ def _deserialize(
                 value["sensor"], None, None
             )
 
+            if sensor is None:
+                raise FMValidationError(f"No sensor found with id {value['sensor']}.")
+
+            # lazy loading now (sensor is somehow not in session after this)
+            sensor.generic_asset
+            sensor.generic_asset.generic_asset_type
+
+            if not self.any_unit and not units_are_convertible(
```

Comment:
The first condition is introduce to stop checking if a unit is convertible when the to_unit is `/x`, right?

I wonder if we could introduce some of the logic to `units_are_convertible` to check for this. If it's too much work, we can create an Issue.

Reply:
I resolved this otherwise, by initializing the `SensorIdField` without a `unit` in case `self.any_unit is True`.